dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/fat/src/com/ibm/ws/fat/wc/tests/WCRequestAutoDecompressTest.java

```diff
@@ -56,7 +58,7 @@ public class WCRequestAutoDecompressTest {
     @BeforeClass
     public static void setUp() throws Exception {
         LOG.info("Setup server: servlet40_requestAutoDecompress");
-        ShrinkHelper.defaultDropinApp(server, APP_NAME + ".war", "/resources");
+        ShrinkHelper.defaultDropinApp(server, APP_NAME + ".war", "decompression.servlets");
```

Comment:
Did we not have `/resources` and so this can be removed without issues?

Reply:
Resources are added automatically, so it wasn't really needed. This third argument accepts packages.

https://github.com/OpenLiberty/open-liberty/blob/8101bceb22cbd3735b9c6ab1e2dbf49b659e53d6/dev/fattest.simplicity/src/com/ibm/websphere/simplicity/ShrinkHelper.java#L461